design/baremetal-operator/bmh_non-bootable_iso.md

```diff
@@ -0,0 +1,73 @@
+# Add non-bootable iso attach API to BareMetalHost
+
+## Status
+
+Implementable
+
+## Summary
+
+Add a new field in BareMetalHost so that it is possible to attach a generic
+non-bootable iso image via Ironic, after the host has been provisioned.
+
+## Motivation
+
+In certain scenarios it is desirable to attach a non-bootable iso image
+to a system after provisioning has finished, for example :
+
+As a config data disk to do some post provisioning configuration for the
+system.
+
+For troubleshooting of the node.
```

Comment:
This is unclear, please elaborate or remove.

Reply:
I was thinking the user might want to run some validations and have some tools to debug, collect logs. Does that make sense ?

Reply:
The ISO is not bootable, right? So you still need access to the operating system to run any tools from it. If you have it, why do you need an ISO?